docs/getting-started.md

```diff
@@ -6,10 +6,10 @@ You can check if your system meets the requirements by running
 `firecracker/tools/devtool checkenv`.
 
 An opinionated way to run Firecracker is to launch an
-[EC2](https://aws.amazon.com/ec2/) `i3.metal` instance with Ubuntu 18.04.
+[EC2](https://aws.amazon.com/ec2/) `i3.metal` instance with Ubuntu 22.04.
```

Comment:
If you do another revision, maybe we should suggest a c5.metal instead of an i3.metal, as it's cheaper, and we don't test on i3 anyway.

Reply:
Makes sense, I've updated it from `i3` to `c5`.